easybuild/easyconfigs/f/FlashPCA2/FlashPCA2-2.0-GCC-10.2.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MakeCp'
+
+name = 'FlashPCA2'
+version = '2.0'
+
+homepage = 'https://github.com/gabraham/flashpca'
+description = """FlashPCA performs fast principal component analysis (PCA) of single nucleotide polymorphism (SNP) data."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/gabraham/flashpca/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_CPPFLAGS.patch']
+
+dependencies = [
+    ('Boost', '1.74.0'),
+    ('Eigen', '3.3.8'),
+    ('SpectrA', '0.9.0'),
+    ('Boost', '1.74.0'),
+    ('PLINK2', '20210701-linux_x86_64', '', True),
+]
+
+buildopts = 'CXXFLAGS="$CXXFLAGS -DSPECTRA"'
```

Comment:
You're redefining the full `CXXFLAGS` value that should be used `make` here, which means you're overwriting where it's added `-DVERSION=...` (see https://github.com/gabraham/flashpca/blob/master/Makefile#L40).

Also, you need to add `-fpermissive` too, to fix this issue:

```
error: invalid conversion from const double* to double* [-fpermissive]
```

Try this instead:

```python
prebuildopts = 'export CXXFLAGS="$CXXFLAGS -fpermissive -DSPECTRA" && '